src/app/styles/components/_carousel.less

```diff
@@ -3,6 +3,7 @@
   > .item {
     img {
       margin: 0 auto;
+      width:100%;
```

Comment:
We had some discussion about whether or not to add this declaration. Pros: it will look better when images slightly smaller than the container are uploaded. Cons: it will look worse (pixelated) when images significantly smaller than the container are uploaded. /shrug